shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js

```diff
@@ -45,6 +52,21 @@ describe('sanitizePdfInteractor', () => {
     ).toBe(mockKey);
   });
 
+  it('calls setPdfFields when PDF has form fields', async () => {
```

Comment:
I really like this test name! Does this mean that the call to `applicationContext.getUseCaseHelpers().setPdfFields(fields)` should be moved into the if check on line 42?

Reply:
Good call!